cinema/models.py

```diff
@@ -84,16 +88,21 @@
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
+    @property
+    def full_name(self):
+        return {"row": self.row, "seat": self.seat}
```

Comment:
The `full_name` property in the `Ticket` model is misleadingly named, as it returns a dictionary with 'row' and 'seat' keys. Consider renaming this property to something more descriptive, such as `seat_location`, to better reflect its purpose.